src/react/components/ButtonGroup.scss

```diff
@@ -0,0 +1,50 @@
+@import '../stylesheets/variables';
+
+.ButtonGroup {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+
+  .GroupedButton {
+    font-size: 12px;
+    line-height: 14px;
+    padding-top: 8px;
+    padding-bottom: 8px;
+    text-align: center;
+    border: 1px solid $light-gray;
+    border-left-width: 0;
+    border-right-width: 0;
+  
+    &:first-child {
+      border-left-width: 1px;
+      border-top-left-radius: 5px;
+      border-bottom-left-radius: 5px;
+    }
+  
+    &:last-child {
+      border-right-width: 1px;
+      border-top-right-radius: 5px;
+      border-bottom-right-radius: 5px;
+    }
+  }
+  
+  .GroupedButton--inactive + .GroupedButton--inactive {
+    border-left-width: 1px;
+  }
+  
+  .GroupedButton--active + .GroupedButton--inactive {
+    border-left-width: 0;
+  }
+  
+  .GroupedButton.GroupedButton--active {
+    font-weight: bold;
+    color: #2C84BC;
+    border: 1px solid #2C84BC;
+    background: #F3F9FD;
+  }
+  
+  .GroupedButton.GroupedButton--inactive {
+    color: #5E5D5D;
```

Comment:
@developish  shouldn't we be using color variables?  I'm scared of trying to keep track of hex values throughout all of our files?

Reply:
I think that as we move to a more component based way of doing things, there's less need to worry about keeping track of these things. Also, these colors are pretty specific to this components, so if I were to extract them they would just end up being named things like `$button-group-active-border` or something and just moving the place where we put them. I'm also not opposed to a future extraction, for instance I'm going to do a custom styling of the select box which will then make sharing them as UI colors make more sense.